### ISSUE TITLE
Dryad spawn rate is too high

### ISSUE THREAD

**Issue by warmer-autumn** [Shorepion/Magica-Infinitum#29]
*Thank you for reporting an issue!
Please make sure to give answers for the following:*

## 1) Which Modpack and version are you using?
Magica Infinitum IV (0.0.7-BETA)
 
## 2) Link to the crashlog (found in the crash-reports folder).
 *Don't post whole crashlogs directly in here!
 I recommend to use pastebin for the crashlogs: https://pastebin.com/*
No crashes, just annoying.
 
## 3) Detailed description of the issue and if possible pls attach images
Dryads spawn constantly. Doesn't matter where, doesn't matter when.

With the dryads come grass, since they bonemeal a 3x3 area around them like every second, resulting in tall grass, flowers, and lots of natural clutter. This severely impacts farming, exploration, building, or literally anything that isn't in a grassy biome.
![2023-02-22_13 49 18](https://user-images.githubusercontent.com/91571521/220742999-7ab8ae2b-0fc1-49b2-804c-76ddb148b0b2.png)
![2023-02-22_13 50 13](https://user-images.githubusercontent.com/91571521/220743001-9c126e49-9f68-4f36-927b-a3f48c63730f.png)

This spawn weight might also be causing issues with overworld spawns.

 
## 4) Is the issue repeatable?
YES.
Just leave a server running lol.

## 5) Do you know a possible fix for the issue?
Remove Ars Magica Legacy or implement a mod/resource pack to manage spawn rates.
The creator themselves said they won't implement a config for spawn weights.

Also, isn't Mana & Artifice a successor to Ars Magica? Seems redundant to have in a modpack.

**Comment by Shorepion** [comment]
Agreed, I wasn't really aware of this. Will investigate and test to get the spawning rate under control.

Ars Magica Legacy was mainly added due to the old bosses that are not in MNA. I will do some more testing concerning mod interaction/coexistance balance. If there will be too much things cancelling each other between the two mods, one (most probably Ars Magica Legacy, will be removed or optional enableable.

Thanks for your report!

**Comment by warmer-autumn** [comment]
Also worth mentioning that I noticed craft tweaker recipes between Ars Magica: Legacy and Mana&Artifice wakeblooms, but none for Chimerite or Vinetum which hinders progression through the latter mod.

Adding some craft tweaker recipes for essential materials and then tweaking oregen would be helpful.

**Comment by Shorepion** [comment]
Closing due to uncontinueing modpack development (Spoiler: Magica Infinitum V for 1.20.1 is in the making...)
